library/core/src/iter/sources/successors.rs

```diff
@@ -1,11 +1,16 @@
 use crate::fmt;
 use crate::iter::FusedIterator;
 
-/// Creates a new iterator where each successive item is computed based on the preceding one.
+/// Creates an iterator which, starting from an initial item,
+/// computes each successive item from the preceding one.
 ///
-/// The iterator starts with the given first item (if any)
-/// and calls the given `FnMut(&T) -> Option<T>` closure to compute each item’s successor.
-/// The iterator will yield the `T`s returned from the closure.
+/// The iterator stores an optional item (`Option<T>`) and a successor closure (`impl FnMut(&T) -> Option<T>`).
+/// The iterator's `next` method returns the stored optional item and
```

Comment:
```suggestion
/// It's `next` method returns the stored optional item and
```

Reply:
I believe this should be "Its", but sure, yes.

Reply:
😃 yes